hyperspy/tests/signal/test_hologram_image.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+#  Copyright 2007-2016 The HyperSpy developers
+#
+# This file is part of  HyperSpy.
+#
+#  HyperSpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+#  HyperSpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
+
+import numpy as np
+
+import numpy.testing as nt
+
+import hyperspy.api as hs
+from scipy.interpolate import interp2d
+
+
+class TestCaseHologramImage(object):
+
+    def setUp(self):
```

Comment:
The tests concern only single images, could you modify it to test the multidimensional capabilities by e.g. reconstructing a (2, 3| x, y) nd-image?